StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentOption+Images.swift

```diff
@@ -91,7 +91,7 @@ extension STPPaymentMethod {
         case .SEPADebit:
             return Image.carousel_sepa.makeImage().withRenderingMode(.alwaysOriginal)
         case .link:
-            return (LinkUI.useNewBrand ? Image.link_logo : Image.pm_type_link_deprecated).makeImage().withRenderingMode(.alwaysOriginal)
+            return Image.link_logo.makeImage()
```

Comment:
There is a few in this PR where we did not need `makeImage()` previously but now we do. I'm not seeing how this complied before..

Reply:
This was one expression:
`(LinkUI.useNewBrand ? Image.link_logo : Image.pm_type_link_deprecated)`
Which then had `.makeImage()` called on it. Sorry for the ambiguity!

Reply:
You should keep `.withRenderingMode(.alwaysOriginal)`, we don't want it to render as a template.

Reply:
Oh I missed the parenthesis! Thanks!